styles/header.css

```diff
@@ -0,0 +1,109 @@
+html {
+    font-family: sans-serif;
+    font-size: 14px;
+    color: #2c2b25;
+}
+
+body {
+    width: 100%;
+    margin: 0;
+}
+
+.top-nav {
+    display: flex;
+    flex-direction: row;
+    padding: 2.5vw;
+}
+
+.logo {
+    width: 100%;
+    height: auto;
+    max-width: 18vw;
```

Comment:
Why are you still adding width 100% if you have a max-width set which is lower than 100%?